workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GetItemCommandOutput } from '@aws-sdk/client-dynamodb';
+import { AwsService, resourceTypeToKey, uuidWithLowercasePrefix } from '@aws/workbench-core-base';
+import Boom from '@hapi/boom';
+import Account from '../models/account';
+import CostCenter from '../models/costCenter';
+import CreateCostCenter from '../models/createCostCenter';
+import AccountService from './accountService';
+
+export default class CostCenterService {
+  private _aws: AwsService;
+  private readonly _tableName: string;
+
+  public constructor(constants: { TABLE_NAME: string }) {
+    const { TABLE_NAME } = constants;
+    this._tableName = TABLE_NAME;
+    this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+  }
+
+  public async getCostCenter(costCenterId: string): Promise<CostCenter> {
+    // Get by id
+    const response = (await this._aws.helpers.ddb
+      .get({
+        pk: costCenterId,
+        sk: costCenterId
+      })
+      .execute()) as GetItemCommandOutput;
+
+    if (response.Item === undefined) {
+      throw Boom.notFound(`Could not find cost center ${costCenterId}`);
+    }
+
+    console.log(`before ${JSON.stringify(response.Item)}`);
+
+    delete response.Item.pk;
```

Comment:
Since we're going to be deleting `pk` and `sk` a lot, I'm thinking we should add a util function in `workbench-core/base/utilities/ddbUtil.ts`.

In that new file, we can define a method like `cleanDDBResponse`, where it takes in a response object and delete `pk`,`sk`, and `dependency`. Thoughts?

Reply:
Also might be good to move the build key methods to `ddbutil` if you can...:)

Reply:
I'd propose we hold off on doing this since Marianna is blocked and this satisfies the requirement for building the response. Its definitely less than ideal, but I don't want to hold Marianna up an longer than needed and we can always solve this in a PR immediately following this one.